misc/swagger-ballerina/modules/ballerina-to-swagger/src/main/java/org/ballerinalang/ballerina/swagger/convertor/service/SwaggerConverterUtils.java

```diff
@@ -119,7 +122,51 @@ public static String generateSwaggerDefinitions(String ballerinaSource, String s
     
         return swaggerSource;
     }
-    
+
+
+    /**
+     * This method will generate open API 3.X specification for given ballerina service. Since we will need to
+     * support both swagger 2.0 and OAS 3.0 it was implemented to convert to swagger by default and convert it
+     * to OAS on demand.
+     *
+     * @param ballerinaSource ballerina source to be converted to swagger/OAS definition
+     * @param serviceName specific service name within ballerina source that need to map OAS
+     * @return Generated OAS3 string output.
+     * @throws IOException When error occurs while converting, parsing input source.
+     */
+    public static String generateOAS3Definitions(String ballerinaSource, String serviceName) throws IOException {
+        // Get the ballerina model using the ballerina source code.
+        BFile balFile = new BFile();
+        balFile.setContent(ballerinaSource);
+        //Create empty swagger object.
+        Swagger swaggerDefinition = new Swagger();
+        BLangCompilationUnit topCompilationUnit = SwaggerConverterUtils.getTopLevelNodeFromBallerinaFile(balFile);
+        String httpAlias = getAlias(topCompilationUnit, "ballerina.net.http");
```

Comment:
can we declare these package names as a const? they have been used in few other places. this is just a suggestion.